### ISSUE TITLE
how to use BranchApiRestClient

### ISSUE THREAD

**Issue by tianweipeng** [uwolfer/gerrit-rest-java-client#230]

**Comment by uwolfer** [comment]
@tianweipeng What would you like to know exactly? Do you have a code sample which does not work as expected?

**Comment by tianweipeng** [comment]
> @tianweipeng What would you like to know exactly? Do you have a code sample which does not work as expected?

I copied the demon code, but branch element returns null when I used gerritApi.projects().list().get() or gerritApi.projects().list().getBranch()

**Comment by uwolfer** [comment]
@tianweipeng Do other API calls work against the same Gerrit server?

**Comment by tianweipeng** [comment]
> @tianweipeng Do other API calls work against the same Gerrit server?

@uwolfer No, this function  "GET /projects/testproject/tags?m=v2 HTTP/1.0" didn't been implemented.

**Comment by uwolfer** [comment]
@tianweipeng Can you please try to call the Gerrit REST API via curl or something like that? It sounds to me like the REST API is not available under the path requested.

**Comment by tianweipeng** [comment]
That REST API is  available.
